molecule/default/molecule.yml

```diff
@@ -10,10 +10,13 @@ driver:
 verifier:
   name: ansible
 provisioner:
+  env:
+    ANSIBLE_STDOUT_CALLBACK: yaml
```

Comment:
While I appreciate the change, I believe it's not linked to the snap installation, so that should go into a separate commit / PR